pkg/beholder/pb/example.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/beholder/pb";
+
+package pb;
+
+// Used for testing 
+message CustomMessage {
```

Comment:
If it is only used for testing, does it make sense to also call it "TestOnlyMessage"?

Reply:
@pavel-raykov  addressed [here](https://github.com/smartcontractkit/chainlink-common/pull/694/commits/43b59453daa04efffbbecd830856e921dd50b066)